### ISSUE TITLE
Application not available

### ISSUE THREAD

**Issue by marccollin** [amonchakai/Hg10#121]
on blackberry market, i don't see Hg10, i have a blackberry passport.

**Comment by rdvl97** [comment]
Same issue here. HG10 appears to have been removed from Blackberry world again.

**Comment by amonchakai** [comment]
Yes, it just go on and off with my patience toward the community. I am tiered to give my own time for free and getting bad reviews. So I am done.

There are builds in the release section if you are interested in Hg10

**Comment by rdvl97** [comment]
Sorry to hear that, man.  :(  
I wish you better luck in your future projects.

**Comment by kenzo44** [comment]
Hi @amonchakai, 

Its developers like you that power the BlackBerry ecosystem. Hg10 is an awesome app.

I would like to help you with developing Hg10...What are some of the bugs you have encountered while developing  Hg10?

So far I am optimizing most of your file structure and will add a pull request soon. I hope I can encourage you to continue developing Hg10.

**Comment by amonchakai** [comment]
Hi ... 

Thats is nice. 

For the problems, just go to the list of issues on github and pick the one you like.
- The main complains are about keeping the connection live. Maybe you can update the QXMPP library. I am not really expecting that will solve the problem. But you can try.
- What would be nice to fix, is the problem with OS update which remove Hg10 from the Hub. There are functions to reload the entry from the hub after an OS update, but somehow just freeze the headless 

Appart that nothing special, fetch the right timestamp when loading the messages from Google would be nice.
